devs/docs/basics.rst

```diff
@@ -152,7 +152,7 @@ Checkstyle
 If you use IntelliJ, there is a Checkstyle plugin available which lets you check
 Checkstyle compliance from within the IDE.
 
-The Checkstyle plugin enforces rules defined in `<PROJECT_ROOT>/gradle/checkstyle/rules.xml`.
+The Checkstyle plugin enforces rules defined in `<PROJECT_ROOT>/gradle/checkstyle/checkstyle.xml`.
```

Comment:
minor: why new name here, because of `configDirectory` change below or just better naming?

Reply:
`checkstyle.xml` is the default name of the files containing the rules, so yes, we can only define `configDirectory` for the gradle plugin, and using a custom name can cause hiccups with the Intellij checkstyle plugin (intend to enable in future so we have a common way to check the style also inside the IDE, without the use of extra files in `idea` directory.

Reply:
@BaurzhanSakhariev see:
![Screenshot_20220622_171740](https://user-images.githubusercontent.com/5058131/175052394-ff5270d5-8938-4c29-a70b-2671093e4fc1.png)

Reply:
I see, thx for clarification!